src/Stores/index.ts

```diff
@@ -207,18 +207,21 @@ export class AppStore implements Store {
     }
   }
   @action async syncAccount() {
-    const { emitter } = textile.account.sync(true)
-    emitter.on('textile.snapshots.found', (item: any) => {
-      const opts: NotificationOptions = {
-        body: item.id,
-        timestamp: moment().unix()
-      }
-      const note = new Notification('Found and applying snapshot', opts)
-    })
+    textile.account.sync(true)
+      .then((stream: ReadableStream) => {
```

Comment:
@carsonfarmer going to move forward with this PR but just FYI this is an update I made to accommodate the new streams. it's worth a new review of these 10 lines